lib/name_drop/resources/base.rb

```diff
@@ -58,6 +58,9 @@ def self.find(client, id)
         else
           response
         end
+      rescue NameDrop::Error => error
+        return if error.detail['code'] == 404
```

Comment:
This appears to be the appropriate place to do this given that other requests don't need to parse out 404 errors.